website/versioned_docs/version-1.0/tutorials/adding-a-module-to-your-runtime.md

```diff
@@ -281,7 +281,7 @@ cargo build --release
 
 ## Next Steps
 
-We wont actually go into the details of testing the Contracts module functionality added to this chain, but if you want to try it out, you can follow our [contract tutorials](https://substrate.dev/tutorials).
+We won't actually go into the details of testing the Contracts module functionality added to this chain, but if you want to try it out, you can follow our [contract tutorials](https://substrate.dev/tutorials).
```

Comment:
I like that you've fixed this error in the current and previous versions of the doc. Did you check thr previous versins fro the other errors you fixed in this pr?

Reply:
Yes, the others appear only once.